### PR TITLE
Validation

### PR DESCRIPTION
**Pull request by kenu21**

### CODE REVIEW

File: src/main/java/com/keniu/controllers/BookController.java [hunk 1]
```diff
@@ -72,7 +73,7 @@ public BookDto save(@RequestBody CreateBookRequestDto createBookRequestDto) {
      */
     @PutMapping("/{id}")
     public BookDto update(@PathVariable Long id,
-            @RequestBody CreateBookRequestDto createBookRequestDto) {
+            @ Valid @RequestBody CreateBookRequestDto createBookRequestDto) {
```
Comment (liliia-ponomarenko) [inline src/main/java/com/keniu/controllers/BookController.java]:
```suggestion
            @Valid @RequestBody CreateBookRequestDto createBookRequestDto) {
```

File: src/main/java/com/keniu/dto/CreateBookRequestDto.java [hunk 2]
```diff
@@ -10,9 +12,13 @@
  */
 @Data
 public class CreateBookRequestDto {
+    @NotBlank
```
Comment (liliia-ponomarenko) [inline src/main/java/com/keniu/dto/CreateBookRequestDto.java:16]: You should also validate the length of the literals. Use the same constraints you use for columns

**Review by liliia-ponomarenko — APPROVED** [review]
Good job! A few minor comments ;)

File: src/main/java/com/keniu/controllers/BookController.java (hunk 1 above)
Comment (kenu21) [reply src/main/java/com/keniu/controllers/BookController.java]: Corrected

File: src/main/java/com/keniu/dto/CreateBookRequestDto.java (hunk 2 above)
Comment (kenu21) [reply src/main/java/com/keniu/dto/CreateBookRequestDto.java:16]: Added @ Size

**Pull request merged**